implants/imix/install_scripts/install_service/main.eldritch

```diff
@@ -205,6 +205,22 @@ def launch_daemon(service_name, executable_path, executable_args):
     sys.shell("launchctl load -w /Library/LaunchDaemons/"+service_name+".plist")
     print("Launch daemon installed")
 
+def windows_service_manager(service_name, service_display_name, service_description, executable_path):
+    create_res = sys.shell("sc.exe create "+service_name+" binpath= "+executable_path+" displayname="+service_display_name+" start= auto type= own")
+    if 'ERROR' in create_res['stdout'] or create_res['stderr'] != "":
+        print("Failed to create service:\n")
+        print(create_res)
```

Comment:
nit: combine these into one print statement

print("failed to create service:\n"+create_rest+"\n")